### ISSUE TITLE
Cannot use SPM

### ISSUE THREAD

**Issue by QuentinBM** [peripheryapp/periphery#190]
Hi,

I had some difficulties integrating Periphery with SPM.

From a fresh project (`swift package init`), I have added the Periphery dependency

```
.package(url: "https://github.com/peripheryapp/periphery", from: "2.0.0")
```

But when I run `swift build`, I have this error:

<img width="1018" alt="Screenshot 2020-11-14 at 13 29 32" src="https://user-images.githubusercontent.com/64783278/99146994-6f990980-267d-11eb-98b0-82f636632ec1.png">

Thanks for your help.

I use swift 5.3
```
Apple Swift version 5.3 (swiftlang-1200.0.29.2 clang-1200.0.30.1)
Target: x86_64-apple-darwin19.6.0
```

**Comment by ileitch** [comment]
Could you show me the contents of your Package.swift?

**Comment by QuentinBM** [comment]
Sure
```
// swift-tools-version:5.3
// The swift-tools-version declares the minimum version of Swift required to build this package.

import PackageDescription

let package = Package(
    name: "test",
    products: [
        .library(
            name: "test",
            targets: ["test"]),
    ],
    dependencies: [
    	.package(url: "https://github.com/peripheryapp/periphery", from: "2.0.0")
    ],
    targets: [
        .target(
            name: "test",
            dependencies: []),
        .testTarget(
            name: "testTests",
            dependencies: ["test"]),
    ]
)
```

**Comment by ileitch** [comment]
I've just released 2.1.0 which should resolve this issue. I've also updated the README, you need to run `swift run periphery` for it be built, since it's not depended upon by any targets.

**Comment by QuentinBM** [comment]
Nice. It just works. ❤️ Thanks for this unbeatable reactivity. 🏃‍♂️💨
